utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -148,11 +148,19 @@ impl PalletCmd {
 		let state_with_tracking = BenchmarkingState::<BB>::new(
 			genesis_storage.clone(),
 			cache_size,
-			self.record_proof,
+			// Record proof size
 			true,
+			// Enable storage tracking
+			true,
+		)?;
+		let state_without_tracking = BenchmarkingState::<BB>::new(
+			genesis_storage,
+			cache_size,
```

Comment:
sounds like `cache_size` and these two boolean args could be turned into a `struct BenchStateConfig { ... }` so that it is explicit and you don't need a comment to say what is what.

Reply:
Yes the code is very horrible. We need to refactor that in the future.